CONTRIBUTING.md

```
@@ -0,0 +1,271 @@
+### Getting Started
+
+1. **Fork the repository** on GitHub
+2. **Clone your fork** locally:
+   ```bash
+   git clone --recursive https://github.com/your-username/HomeKey-ESP32.git
+   cd HomeKey-ESP32
+   ```
+
+### Development Setup
+
+1. **Install ESP-IDF** (see [here](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/get-started/index.html#what-you-need) for details):
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)